packages/assets/README.md

```diff
@@ -20,11 +20,19 @@ Check [assets-backend](https://github.com/orbiting/assets-backend) for a deploya
 
   ENVs: `ASSETS_HMAC_KEY`
 
-- `/render?url=:url&width=:width&height=:height`
-
-  renders :url with a viewport of :width x :hight to a png. webp conversion not supported.
-
-  ENVs: `PHANTOMJSCLOUD_API_KEY`, `RENDER_URL_WHITELIST`: comma separated, accept: :url.indexOf(whiteUrl) === 0, `FRONTEND_BASIC_AUTH_USER`, `FRONTEND_BASIC_AUTH_PASS`
+- `/render?url=:url&[viewport=[:width]x[:height]]&[zoomFactor=:sf]`
+
+  - renders :url
+  - optional :viewport
+    - succeeds deprecated: `width=:width&height=:height`
+    - default 1200x200
```

Comment:
I'd prefer `1200x1`. 200 is arbitrary. If the default behaviour is `auto` height, 1 will lead to the right size in all cases except when there are height checks on the rendered pages. However those checks will most-likely also expect more than 200 height.

Reply:
totally agree